media-video/handbrake/handbrake-1.2.2.recipe

```diff
@@ -35,16 +35,19 @@ REQUIRES="
 	lib:libharfbuzz$secondaryArchSuffix
 	lib:libiconv$secondaryArchSuffix
 	lib:libjansson$secondaryArchSuffix
+	lib:liblzma$secondaryArchSuffix
 	lib:libmkv$secondaryArchSuffix
 	lib:libmp3lame$secondaryArchSuffix
 	lib:libmp4v2$secondaryArchSuffix
 	lib:libmpeg2$secondaryArchSuffix
 	lib:libopus$secondaryArchSuffix
 	lib:libsamplerate$secondaryArchSuffix
+	lib:libspeex$secondaryArchSuffix
 	lib:libtheora$secondaryArchSuffix
 	lib:libvorbis$secondaryArchSuffix
 	lib:libvpx$secondaryArchSuffix
 	lib:libx264$secondaryArchSuffix
+	lib:libx265$secondaryArchSuffix
```

Comment:
Required? It is disabled in the configure invocation.

Reply:
Not required for current build, but will enable/require after the x265 3.x integration.

Reply:
Fixed.